packages/modular-scripts/src/build.ts

```diff
@@ -250,6 +256,7 @@ async function makeBundle(
     input: path.join(packagesRoot, directoryName, packageJson.main),
     external: (id) => {
       // via tsdx
+      // TODO: this should probably be included into deps instead
```

Comment:
I think we should start up for a strategy for managing all the TODOs - either a central file we can view them all or extract them into issues (or both linked?)

Reply:
These are non critical TODOs so I've left them inline. Happy to organise them later.